sig-scheduling/annual-report-2021.md

```diff
@@ -87,17 +91,18 @@ In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $Y
 Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
 -->
 
-New in $YYYY:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
+New in 2021:
+- [kube scheduler simulator](https://github.com/kubernetes-sigs/kube-scheduler-simulator)
 -
 
-Retired in $YYYY:
-- [$subproject-name](https://git.k8s.io/community/$sig-id#$subproject-name)
+Retired in 2021:
+- None
```

Comment:
@liggitt @Huang-Wei what should we do with the following subprojects, no activity for a while on them:
- https://github.com/kubernetes-sigs/poseidon
- https://github.com/kubernetes-sigs/kube-batch (this is now part of volcano, should we keep it?)

Reply:
I propose we remove them from https://github.com/kubernetes/community/blob/master/sig-scheduling/README.md#subprojects since they are not active and none of their owners show up to sig meetings, does that make sense?

Reply:
Actually these 2 projects have been inactive a long time, at least since I composed the 2020 annual report. I think I should archive them, and then remove them from scheduling README. Created two issues:

- https://github.com/kubernetes-sigs/kube-batch/issues/959
- https://github.com/kubernetes-sigs/poseidon/issues/187

Reply:
listing those as `Continuing` for the 2021 report with a comment/link to the issue proposing archiving in 2022 sounds good

Reply:
listed them, we will mark them as retired in next year report.